app/controllers/users_controller.rb

```diff
@@ -49,6 +52,17 @@ def enroll
   end
 
   ####################################################
+  # User listing page for Admins                     #
+  ####################################################
+  def index
+    @users = if params[:term].present?
+               User.search(params[:term])
+             else
+               User.all
```

Comment:
This is slow and not super useful. for the default, I suggest either `User.none` or a query for the last 20 users where permissions equal `User::Permissions::INSTRUCTOR`.